umpleonline/scripts/umpleCollab.js

```diff
@@ -7,24 +7,29 @@
 // It is responsible for connecting to the UmpleCollabServer, sending and receiving updates,
 // and handling inactivity
 
-Collab = new Object();
-let sockett= null;
-let dc = null;
-let baseclientID = null;  
-let attempt = 0;
-let checktemp = false;
-let isConnected = false;
-let inactivityDisabled = false;
+Collab = new Object();  // Object to store the functions for collaboration
+let oldSocket= null; // Variable to store the socket connection
+let packetLoss = null; // Variable to store the number of packet loss
+let baseclientID = null;  // Variable to store the client ID
+let attempt = 0;   // Variable to store the number of attempts
+let checktemp = false;  // Variable to check if the model is temporary
+let isConnected = false; // Variable to store the connection status
+let inactivityDisabled = false; // Variable to store the inactivity status
 
 // the main process of Idle timer
-let inactivityTime = 600000; // 10 minutes
-let inactivitywarningTime = 540000; // 9 minutes
+let defaultInactivityTime = 600000; // 10 minutes
```

Comment:
@payamd all constants can be declared using **const** keyword instead of let

Reply:
Sure, I will do that. We can also read these variables from the config file if you want, as you mentioned, but I think reading some variables from the config file can make it more complex.